libs/blocks/icon-block/icon-block.js

```diff
@@ -57,7 +67,7 @@ function decorateContent(el) {
     }
     const size = getBlockSize(el, 2);
     const variant = [...variants].filter((v) => el.classList.contains(v))?.[0] ?? 'fullwidth';
-    decorateBlockText(el, iconBlocks[size][variant]);
+    decorateBlockText(el, isUpAndInline ? ['xs', 's'] : iconBlocks[size][variant]);
```

Comment:
Can we instantiate this ternary somewhere else? I don't think it being a param is particularly clear.

Reply:
`const textSize = upAndInline(el) ? ['xs', 's'] : iconBlocks[size][variant];`
`decorateBlockText(el, textSize);`
Something like this ?